lib/utils/dialogs.js

```diff
@@ -0,0 +1,23 @@
+const {dialog} = require('electron').remote;
```

Comment:
this file is exactly the same as `app/utils/dialogs.js` or? 🤔

Reply:
Kind of, yeah. We did that because the closure is suspended in different processes when we're a closing a single tab vs closing all the windows at once (one is on electron's main process, the other on the renderer process). The was the simplest way we found to go around this (different files with different electron includes), but there's definitely a better way, sure.

Reply:
@albinekb @chabou What is the current situation?

Reply:
Interesting @antonioalmeida, do you think we should add a comment to the top of both files to describe that (why there's two copies of the file) so that the next person coming along have a chance to understand what's going on 🤔 💭  😄

Reply:
Sure thing 👍